frontend/packages/kubevirt-plugin/src/components/modals/support-modal/support-modal.tsx

```diff
@@ -46,14 +49,17 @@ const SupportModal: React.FC<SupportModalProps> = ({ onConfirm, close, community
           ) : (
             <>
               <StackItem>
-                {t('kubevirt-plugin~This template does not have a support statement defined.')}
+                {t(
+                  'kubevirt-plugin~The support level is not defined in the template yaml. To mark this template as supported, add these two annotations to the template yaml:',
```

Comment:
```suggestion
                  'kubevirt-plugin~The support level is not defined in the template yaml. To mark this template as supported, add these two annotations to the template:',
```
the reason for removing the explicit YAML suggestion is because we can edit annotations also from details view

Reply:
could we remove YAML and replace it with "add these two annotations in the template details"?

Reply:
msg updated